independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1001 @@
+                        },
+                        "url": {
+                          "type": "string",
+                          "description": "actor url"
+                        }
+                      },
+                      "description": "actor"
+                    },
+                    "generator": {
+                      "type": "object",
+                      "properties": {
+                        "id": {
+                          "type": "string",
+                          "description": "generator id"
+                        },
+                        "displayName": {
+                          "type": "string",
+                          "description": "generator displayName"
+                        },
+                        "displayNameKey": {
+                          "type": "string",
+                          "description": "generator displayNameKey"
+                        },
+                        "type": {
+                          "type": "string",
+                          "description": "generator type"
+                        },
+                        "avatarUrl": {
+                          "type": "string",
+                          "description": "generator avatarUrl"
+                        },
+                        "url": {
+                          "type": "string",
+                          "description": "generator url"
+                        }
+                      },
+                      "description": "generator"
+                    },
+                    "cause": {
+                      "type": "object",
+                      "properties": {
+                        "id": {
+                          "type": "string",
+                          "description": "cause id"
+                        },
+                        "displayName": {
+                          "type": "string",
+                          "description": "cause displayName"
+                        },
+                        "displayNameKey": {
+                          "type": "string",
+                          "description": "cause displayNameKey"
+                        },
+                        "type": {
+                          "type": "string",
+                          "description": "cause type"
+                        },
+                        "avatarUrl": {
+                          "type": "string",
+                          "description": "cause avatarUrl"
+                        },
+                        "url": {
+                          "type": "string",
+                          "description": "cause url"
+                        }
+                      },
+                      "description": "cause"
+                    },
+                    "extraData": {
+                      "type": "string",
+                      "description": "extraData"
+                    }
+                  },
+                  "description": "historyMetadata"
```

Comment:
@Flow-Joe Can you please fix the grammatical mistakes of description values.

Documentation - [Description values of properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the description values in swagger

Reply:
Fixed grammatical errors.